dashboard/config/linux/upstream-arm-full.config

```diff
@@ -1521,11 +1535,8 @@ CONFIG_BRIDGE_EBT_LOG=y
 CONFIG_BRIDGE_EBT_NFLOG=y
 CONFIG_IP_SCTP=y
 # CONFIG_SCTP_DBG_OBJCNT is not set
-CONFIG_SCTP_DEFAULT_COOKIE_HMAC_MD5=y
-# CONFIG_SCTP_DEFAULT_COOKIE_HMAC_SHA1 is not set
-# CONFIG_SCTP_DEFAULT_COOKIE_HMAC_NONE is not set
-CONFIG_SCTP_COOKIE_HMAC_MD5=y
-CONFIG_SCTP_COOKIE_HMAC_SHA1=y
+# CONFIG_SCTP_DEFAULT_COOKIE_HMAC_SHA256 is not set
+CONFIG_SCTP_DEFAULT_COOKIE_HMAC_NONE=y
```

Comment:
Upstream is changing CONFIG_SCTP_DEFAULT_COOKIE_HMAC_MD5 to CONFIG_SCTP_DEFAULT_COOKIE_HMAC_SHA256.

I wanted to reflect this change updating the config. Probably something went wrong.
Let's enable CONFIG_SCTP_DEFAULT_COOKIE_HMAC_SHA256?